md-dao/src/main/java/com/wipro/ats/bdre/md/dao/JobDAO.java

```diff
@@ -305,29 +303,26 @@
                             BatchConsumpQueue batchConsumpQueueBSUpdated = (BatchConsumpQueue) updateBatchStatusCriteria.list().get(0);
                             batchConsumpQueueBSUpdated.setBatchStatus(accessedBatchStatus);
                             session.update(batchConsumpQueueBSUpdated);
-                            //BCQObjectList.add(batchConsumpQueue);
                         }
-                        //queueIdMap.put(blankBCPProcess.getProcessId(),BCQObjectList);
-
                     }
                 }
 
-                Criteria notNullBCPCriteria = session.createCriteria(Process.class).add(Restrictions.eq("process", parentProcess))
-                        .add(Restrictions.eq("deleteFlag", false)).add(Restrictions.ne("enqueuingProcessId", 0))
-                        .add(Restrictions.isNotNull("batchCutPattern"));
+                Criteria notNullBCPCriteria = session.createCriteria(Process.class).add(Restrictions.eq(PROCESS, parentProcess))
+                        .add(Restrictions.eq(DELETEFLAG, false)).add(Restrictions.ne(ENQUEUINGPROCESSID, 0))
+                        .add(Restrictions.isNotNull(BATCHCUTPATTERN));
                 for (Object notNullBCPObject : notNullBCPCriteria.list()) {
                     Long sourceBatchId = null;
                     Process notNullBCPProcess = (Process) notNullBCPObject;
                     String batchCutPattern = notNullBCPProcess.getBatchCutPattern();
                     Criteria likeBCPCriteria = session.createCriteria(BatchConsumpQueue.class).add(Restrictions.like("batchMarking", "%" + batchCutPattern + "%"))
-                            .add(Restrictions.eq("process", notNullBCPProcess)).addOrder(Order.asc("batchBySourceBatchId")).setMaxResults(1);
+                            .add(Restrictions.eq(PROCESS, notNullBCPProcess)).addOrder(Order.asc(BATCHBYSOURCEBATCHID)).setMaxResults(1);
                     if (likeBCPCriteria.list().size() > 0) {
                         BatchConsumpQueue batchConsumpQueue = (BatchConsumpQueue) likeBCPCriteria.list().get(0);
                         sourceBatchId = batchConsumpQueue.getBatchBySourceBatchId().getBatchId();
                     }
 
                     Criteria updateTargetBatchCriteriaWithBM = session.createCriteria(BatchConsumpQueue.class)
-                            .add(Restrictions.le("batchBySourceBatchId", sourceBatchId)).add(Restrictions.eq("process", notNullBCPProcess));
+                            .add(Restrictions.le(BATCHBYSOURCEBATCHID, sourceBatchId)).add(Restrictions.eq(PROCESS, notNullBCPProcess));
 
                     for (Object updateBCQObject : updateTargetBatchCriteriaWithBM.list()) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)